.env.example

```diff
@@ -1,4 +1,7 @@
 # ADD TO CONFIGURE HOSTED AZURE AD FOR AUTH
+AWS_REGION=eu-central-1
+AWS_ACCESS_KEY_ID=test
+AWS_SECRET_ACCESS_KEY=test
```

Comment:
As these are not secrets, could we default them in the per app environment context? e.g. 
`${AWS_SECRET_ACCESS_KEY:-test}`
This will mean the values are overridable but no need to update .env for existing users if they don't need to override this.